htmlhint-server/src/server.ts

```diff
@@ -79,14 +80,19 @@ function makeDiagnostic(problem: htmlhint.Error, lines: string[]): server.Diagno
  */
 function getConfiguration(filePath: string): any {
     var options: any;
-    if (settings.htmlhint && settings.htmlhint.options && Object.keys(settings.htmlhint.options).length > 0) {
-        options = settings.htmlhint.options;
-    }
-    else {
+
+    if (!settings.htmlhint) {
         options = findConfigForHtmlFile(filePath);
     }
+    else if (settings.htmlhint.configFile) {
```

Comment:
I'd move this around so that you have something like:

```
if (settings.htmlhint.configFile && fileExists(settings.htmlhint.configfile)) {
  // use it
}
else {
  if (settings.htmlhint.configFile) {
    // emit warning that file couldn't be found
  }
  options = findConfigForHtmlFile(filePath)
}

Reply:
@plumpNation any news ?